pkg/vpc/vpc.go

```diff
@@ -470,25 +469,26 @@ func SelectNodeGroupSubnets(nodegroupAZs, nodegroupSubnets []string, subnets api
 	if len(subnets) < numNodeGroupsAZs || len(subnets) < numNodeGroupsSubnets {
 		return nil, fmt.Errorf("VPC doesn't have enough subnets for nodegroup AZs or subnets %s", makeErrorDesc())
```

Comment:
does `makeErrDesc` need to be a function? Looks like its a static error message (I know this isn't related to the PR, but looks like a good opportunity to clean up 😄 )

Reply:
Ah missed this, yeah looks like a bit of a premature optimization making it lazy.